### PR TITLE
Display a message if there are no reminders

### LINKED ISSUE #5: Display a message if there are no reminders
Update the app to have a conditional that will only display a reminder if reminders exist. In other words, if there are existing reminders the user should see the most recent reminder. If there are no reminders, the user should see a message to add a new note, with a link or button that routes them to `/reminders/new`.

### PR DESCRIPTION
**Pull request by becs919**
@martensonbj @Tman22 
Added a if conditional to reminders.hbs
Temporarily took out the hardcoded server data to test functionality
Added message that displays to users when there are no reminders
*Will push up test next but closes #5 

## Purpose

Display a message for users when there are no reminders. Add a button that links them to reminders/new

## Approach

Added a if/else conditional 

### Learning

https://guides.emberjs.com/v2.2.0/templates/conditionals/

### Test coverage 

Working on next

### Follow-up tasks

acceptance tests and issue #6

### CODE REVIEW

**Comment by becs919** [comment]
@Tman22 @martensonbj 

We wrote two tests for this issue. Closes #5

File: app/templates/reminders.hbs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <h1>remEMBER APRIL O'NEIL RULES!</h1>
+{{#if model}}
 <div class='reminders-div'>
```
Comment (martensonbj) [inline app/templates/reminders.hbs]: let's indent the stuff within the if block

File: app/templates/reminders.hbs [hunk 2]
```diff
@@ -7,7 +8,15 @@
   {{/each}}
 </div>
 {{#link-to 'reminders.new' class='new-button'}}
-  <button>Create New Reminder</button>
+<button>Create New Reminder</button>
 {{/link-to}}
+{{else}}
+<div>
```
Comment (martensonbj) [inline app/templates/reminders.hbs]: see comment above - indent the stuff within the conditional blocks

File: app/templates/reminders.hbs [hunk 3]
```diff
@@ -7,7 +8,15 @@
...
+<p class="no-reminders">You have no reminders! :-(((</p>
+{{#link-to 'reminders.new' class='new-button'}}
+<button>Create One Here</button>
```
Comment (martensonbj) [inline app/templates/reminders.hbs]: indent please

**Review by martensonbj — CHANGES_REQUESTED** [review]
quick indentation clean up

**Comment by becs919** [comment]
@Tman22 @martensonbj 

Fixed the indents on the conditional.

**Comment by martensonbj** [comment]
👍

**Pull request merged**